ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -734,7 +735,9 @@ public void set3scaleEnvVar(String value) {
             fail("Syndesis server did not start in 300s with new variable", e);
         }
 
-        WebDriverRunner.getWebDriver().navigate().refresh();
+        // new server pod means we need to reset rest route url
+        RestUtils.updateRestUrl(true);
```

Comment:
it should be necessary only to call `RestUtils.reset()` 

next time the call to `RestUtils.getRestUrl()` would create a new port forward anyway, so the `updateRestUrl()` is not needed imho

Reply:
or just make this static https://github.com/syndesisio/syndesis-qe/blob/289cbd973191fe14087cb4fc82bad3f5ee5d0ed3/utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java#L211-L215